bin/cli.js

```diff
@@ -11,9 +11,14 @@ const cli = new Cli()
 program
   .version(pkgJson.version)
   .arguments('<cmd>')
-  .action((cmd) => {
+  .option(
+    '-b, --blackduck <directory>',
```

Comment:
@cstolli I'm not sure we want to do it this way in `pr-bumper`. How do you feel about letting the decision whether to run the (insert generic name for blackduck here) stuff be driven by `.pr-bumper.json`, then projects can configure it on a per project basis, and the build step doesn't need to change to include a different command line switch for different projects.